modules/apps/portal-search/portal-search-web/src/main/resources/com/liferay/portal/search/web/internal/date/facet/portlet/display/template/dependencies/portlet_display_template_checkbox.ftl

```diff
@@ -107,7 +107,7 @@
 							firstDayOfWeek=dateFacetCalendarDisplayContext.getFromFirstDayOfWeek()
 							monthParam="fromMonth"
 							monthValue=dateFacetCalendarDisplayContext.getFromMonthValue()
-							name="${namespace}fromInput"
+							name="fromInput"
```

Comment:
We can include this change, but we should look at a solution that doesn't require removing this namespace. Even if the namespace is added twice in the resulting Liferay.component name (assuming that's the issue why it wasn't able to find the component name), regardless, the logic in `modified_facet.js` and `date_facet.js` should still work.

Reply:
Discussed with @oliv-yu and because the `fromInputName`/`toInputName` is defined in the `view.jsp` (outside of the templates) this is forced to be named properly. And this was added only recently which caused this regression so this should be safe to remove.